eng/generators.targets

```diff
@@ -1,51 +1,56 @@
 <Project>
 
   <PropertyGroup>
+    <!-- Enable LibraryImportGenerator for CoreLib. -->
```

Comment:
I understand that this already existed before this change but it feels weird to special case CoreLib in here. Can we move this property into the shared CoreLib project instead?